### DISCUSSION TITLE
Advantage over comparable products?

### DISCUSSION THREAD

**Discussion by mahjoubkzbr** [omkarcloud/google-maps-scraper#179] [General]
Hey there, really great software u made here! Honestly, it looks great and takes a lot of work of the users shoulders. However, i wonder wether there is an advantage over comparable products like outscraper 🤔

Maybe i'm not deep enough into the topic of lead generation via scraping, but rn it seems to have hardly any advantage apart from being open source. Either way, great work, great software, keep up the good work please!

**Comment by Chetan11-dev** [comment]
Single Most Important, Unlimited Results for a 1 time investment.
